rand_distr/tests/uniformity.rs

```diff
@@ -25,9 +25,13 @@ fn unit_sphere() {
     for _ in 0..N_SAMPLES {
         let v: [f64; 3] = dist.sample(&mut rng);
         for i in 0..N_DIM {
-            histograms[i].add(v[i]).map_err(
-                |e| { println!("v: {}", v[i]); e }
-            ).unwrap();
+            histograms[i]
+                .add(v[i])
+                .map_err(|e| {
+                    println!("v: {}", v[i]);
+                    e
+                })
+                .unwrap();
```

Comment:
I think this decreases readability, but I guess it's good enough.

Reply:
Yeah, I guess you're right. Feel free to post rustfmt config changes / a skip annotation if you prefer.

I'd really like to get the point where we can just run `cargo fmt` on the project so would prefer not to just revert stuff like this — but it may not be until Rustfmt 2.0 that we get there (and even then, we'll have to make further compromises).